src/main/java/Number.java

```diff
@@ -0,0 +1,15 @@
+public class Number {
+
+    private final Integer number;
```

Comment:
참조 자료형 대신 기본 자료형을 사용하면 어떨까요? 아래의 글을 참고하세요.
https://jaehun2841.github.io/2019/03/01/effective-jave-item61

```suggestion
    private final int number;
```